libraries/botframework-streaming/src/utilities/createNodeServer.ts

```diff
@@ -8,39 +8,54 @@
 
 import { INodeServer, INodeSocket } from '../interfaces';
 
-export const createNodeServer = function (callback?: (socket: INodeSocket) => void): INodeServer {
+type ConnectionListener = (socket: INodeSocket) => void;
+
+/**
+ * Create a Node 'net' server
+ *
+ * @param callback Optional connection listener
+ * @returns a Node 'net' server instance
+ */
+export function createNodeServer(callback?: ConnectionListener): INodeServer {
     if (callback && typeof callback !== 'function') {
-        throw new TypeError(`Invalid callback; callback parameter must be a function to create Node 'net' Server.`);
+        throw new TypeError("Invalid callback; callback parameter must be a function to create Node 'net' Server.");
```

Comment:
Is this a lint rule or personal preference? If the latter, we should align on the pattern, codify it and add it to the repo.

Reply:
We should probably eslint this - IMO backticks should be reserved for templating.

Reply:
Filed https://github.com/microsoft/botbuilder-js/issues/3685